nemo/collections/common/prompts/canary.py

```diff
@@ -38,13 +50,9 @@ class CanaryPromptFormatter(PromptFormatter):
             "template": f"{CANARY_BOS}|source_lang||task||target_lang||pnc|",
             "slots": {
                 "source_lang": Modality.Text,
-                "task": Modality.TextLiteral(
-                    "asr", "ast", "translate", "transcribe", "s2t_translation", "<|transcribe|>", "<|translate|>"
-                ),
+                "task": Modality.TextLiteral(*[t for t in (TASK_TRANSLATE | TASK_TRANSCRIBE)]),
```

Comment:
```suggestion
                "task": Modality.TextLiteral(*(TASK_TRANSLATE | TASK_TRANSCRIBE)),
```
should work too, right?